[2024-12-26T08:25:20.319+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-26T08:25:20.389+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transform_forex_data_dag.transform_and_load_forex_data scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2024-12-26T08:25:20.425+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transform_forex_data_dag.transform_and_load_forex_data scheduled__2024-12-25T00:00:00+00:00 [queued]>
[2024-12-26T08:25:20.432+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-26T08:25:20.496+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): transform_and_load_forex_data> on 2024-12-25 00:00:00+00:00
[2024-12-26T08:25:20.511+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-26T08:25:20.515+0000] {standard_task_runner.py:72} INFO - Started process 3243 to run task
[2024-12-26T08:25:20.516+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transform_forex_data_dag', 'transform_and_load_forex_data', 'scheduled__2024-12-25T00:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/Forex_tranformation.py', '--cfg-path', '/tmp/tmp0dnjqr6i']
[2024-12-26T08:25:20.529+0000] {standard_task_runner.py:105} INFO - Job 63: Subtask transform_and_load_forex_data
[2024-12-26T08:25:20.673+0000] {task_command.py:467} INFO - Running <TaskInstance: transform_forex_data_dag.transform_and_load_forex_data scheduled__2024-12-25T00:00:00+00:00 [running]> on host 4fe538676cbc
[2024-12-26T08:25:20.830+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transform_forex_data_dag' AIRFLOW_CTX_TASK_ID='transform_and_load_forex_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-25T00:00:00+00:00'
[2024-12-26T08:25:20.833+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-26T08:25:20.875+0000] {connection.py:414} INFO - Snowflake Connector for Python Version: 3.12.4, Python Version: 3.12.8, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-12-26T08:25:20.877+0000] {connection.py:1197} INFO - Connecting to GLOBAL Snowflake domain
[2024-12-26T08:25:20.879+0000] {connection.py:1278} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
